gpii/node_modules/lifecycleManager/test/js/LifecycleManagerQueueTests.js

```diff
@@ -63,7 +63,7 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
         var lifecycleManager = gpii.lifecycleManager();
         jqUnit.expect(1);
 
-        var queuePromise = lifecycleManager.addToQueue(gpii.tests.lifecycleManager.queue.createCustomTask(1, fluid.identity, 1));
+        var queuePromise = lifecycleManager.addToActionQueue(gpii.tests.lifecycleManager.queue.createCustomTask(1, fluid.identity, 1));
```

Comment:
We need to add corresponding tests now for the other queue, the queue of user logon actions. In particular, somewhere (either in these new tests, or in the existing UserLogonTests) we need to test the headline feature of this branch - that a proximity triggered request for user B when user A is logged on does not cause a temporary intermediate state of "noUser" logged on.